app/code/core/Mage/Core/Model/Design/Package.php

```diff
@@ -933,6 +934,121 @@ protected function _prepareUrl($uri)
         return $uri;
     }
 
+    /**
+     * @param array $allItems
+     * @param array $itemUrls
+     * @param bool $inline
+     * @return string
+     * @throws JsonException
+     */
+    public function renderImportMap(array $allItems, array $itemUrls, bool $inline): string
+    {
+        $importMap = [];
+        $cacheKey = [];
+        $fileHashKey = []; // No timestamps in hash key in production mode
+        $useCache = Mage::app()->useCache('import_map');
```

Comment:
```suggestion
        $useCache = !$inline && Mage::app()->useCache('import_map');
```
No need to use cache for inline mode as rendering is just json_encode which is too fast to bother caching.